.github/CODEOWNERS

```diff
@@ -4296,6 +4296,7 @@
 /types/react-highlight-words/                       @mhegazy @diogodca @kellyrmilligan
 /types/react-highlighter/                           @oizie
 /types/react-holder/                                @isman-usoh
+/types/react-hot-loader/                            @jedmundo
```

Comment:
Don't change this file as is auto-generated. Instead you can be added on the react-hot-loader "definititions by" if you'll like to be a code-owner.

Reply:
Thanks! Done.